app/views/unattended/provisioning_templates/PXEGrub/pxegrub_default_local_boot.erb

```diff
@@ -4,8 +4,17 @@ name: PXEGrub default local boot
 model: ProvisioningTemplate
 %>
 <%# Used to boot provisioned hosts, do not associate or change the name. %>
-
-default=0
+<%
+  # Grub1 only supports numeric default statement, this allows conversion to number. First define
+  # array of directories we will search for EFI booloaders:
+  paths = ["fedora", "redhat", "centos", "debian", "ubuntu", "sles", "opensuse", "Microsoft", "EFI"]
+  # Add remaining entries to it and use this to convert to number:
+  items = paths.push("discovery")
+  # Read default setting but since "local" is missing, use the first path available.
+  default_setting = global_setting("default_pxe_item_global", paths.first).sub("local", paths.first)
+  default_item = items.index(default_setting) || 0
+%>
+default=<%= default_item %>
 timeout=20
 
-<%= snippet "pxegrub_chainload" %>
+<%= snippet "pxegrub_chainload", variables: {paths: paths} %>
```

Comment:
Shouldn't we update `pxegrub_global_default.erb` as well, as it's also using `pxegrub_chainload` snipped (https://github.com/theforeman/foreman/blob/ae96cb70130968bdcc8b7a9a9af7edc7c6e0d9a8/app/views/unattended/provisioning_templates/PXEGrub/pxegrub_global_default.erb#L15)

Reply:
Oh sure. And also updating commuinty-templates PR.